src/target/adiv5.c

```diff
@@ -753,7 +753,7 @@ void adiv5_dp_init(ADIv5_DP_t *dp)
 
 		dp->target_partno = (targetid & ADIV5_DP_TARGETID_TPARTNO_MASK) >> ADIV5_DP_TARGETID_TPARTNO_OFFSET;
 
-		DEBUG_INFO("TARGETID 0x%08" PRIx32 " designer 0x%" PRIx32 " partno 0x%" PRIx32 "\n", targetid,
+		DEBUG_INFO("TARGETID 0x%08" PRIx32 " designer 0x%" PRIx16 " partno 0x%" PRIx16 "\n", targetid,
```

Comment:
This duplicates #1163 but suffers from the same problem as the above format string.

Reply:
I've also switched this to `%x`, and confirmed that it still builds.